pkg/config/machine/machine.go

```diff
@@ -83,32 +83,32 @@ type Device struct {
 // bonded interface.
 type Bond struct {
 	Interfaces      []string `yaml:"interfaces"`
-	ARPIPTarget     []string `yaml:"arpiptarget"`
+	ARPIPTarget     []string `yaml:"arpIPTarget"`
 	Mode            string   `yaml:"mode"`
-	HashPolicy      string   `yaml:"hashpolicy"`
-	LACPRate        string   `yaml:"lacprate"`
-	ADActorSystem   string   `yaml:"adactorsystem"`
-	ARPValidate     string   `yaml:"arpvalidate"`
-	ARPAllTargets   string   `yaml:"arpalltargets"`
+	HashPolicy      string   `yaml:"xmitHashPolicy"`
+	LACPRate        string   `yaml:"lacpRate"`
+	ADActorSystem   string   `yaml:"adActorSystem"`
+	ARPValidate     string   `yaml:"arpValidate"`
+	ARPAllTargets   string   `yaml:"arpAllTargets"`
 	Primary         string   `yaml:"primary"`
-	PrimaryReselect string   `yaml:"primaryreselect"`
-	FailOverMac     string   `yaml:"failovermac"`
-	ADSelect        string   `yaml:"adselect"`
+	PrimaryReselect string   `yaml:"primaryReselect"`
+	FailOverMac     string   `yaml:"failOverMac"`
+	ADSelect        string   `yaml:"adSelect"`
 	MIIMon          uint32   `yaml:"miimon"`
 	UpDelay         uint32   `yaml:"updelay"`
 	DownDelay       uint32   `yaml:"downdelay"`
```

Comment:
Not sure what the kernel version is, but should these be `upDelay` and `downDelay`?

Reply:
Looks like these are good 👍

Reply:
Yeah, for whatever reason, the kernel options run those together as single words.